storio-content-resolver-annotations-processor-test/build.gradle

```diff
@@ -0,0 +1,35 @@
+import org.gradle.internal.jvm.Jvm
+
+apply plugin: 'com.android.library'
+apply from: 'build.workaround-missing-resource.gradle'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+    }
+}
+
+dependencies {
+    testCompile libraries.storIOContentResolver
+    testCompile libraries.storIOContentResolverAnnotationsProcessor
+    testCompile libraries.googleTestingCompile
+    testCompile files(getRuntimeJar())
```

Comment:
Dunno it's not really a library. I mean it doesn't have versioning so moving it to libraries won't give much profit I guess.